docs/slides/content/social-contracts.md

```diff
@@ -144,5 +144,5 @@ _"Can you give me an example of not being open?" could lead to actionable items
 ### Related & Used Practices
 * [Social Contract](https://openpracticelibrary.com/practice/social-contract/)
 * [Affinity Mapping](https://openpracticelibrary.com/practice/affinity-mapping/)
-* [Voting (fist of five, dot voting, ..)](https://openpracticelibrary.com/practice/confidence-voting/)
+* [Voting (fist of five, dot voting)](https://openpracticelibrary.com/practice/confidence-voting/)
```

Comment:
Apologies, I just realized this one. In here, we meant that there are a lot of voting strategies; such as fist of five, dot voting. They are not limited to two. Can we do something like;
```suggestion
* [Voting (fist of five, dot voting, etc)](https://openpracticelibrary.com/practice/confidence-voting/)
```

or please feel free to suggest something more appropriate :)

Reply:
This is fine except you need a period after "etc."

Reply:
I committed this change. 
https://github.com/rht-labs/tech-exercise/pull/98/commits/43a586466cfa05c7ba01288e9496f7bac3199a3b